src/styles/base/_override.typography.scss

```diff
@@ -15,123 +15,113 @@
 .ds-text-heading--lg,
 .ds-text-heading--md,
 .ds-text-heading--sm {
+  font-family: Montserrat;
   font-weight: $font-semibold;
-  font-family: "Montserrat";
-}
-
-.ds-title,
-.ds-text-heading--4xl {
-  font-weight: 300;
 }
 
 .ds-h1,
 .ds-text-heading--3xl {
-  font-size: $h1-mobile-font-size;
-  line-height: 32px;
-
+  font-size: $font-size-2xl;
+  line-height: 40px;
+  margin-bottom: 44px;
   @media screen and (min-width: $width-md) {
-    line-height: 47px;
     font-size: $font-size-3xl;
+    line-height: 48px;
+    margin-bottom: 52px;
   }
 }
 
 .ds-h2,
 .ds-text-heading--2xl {
-  margin-top: 36px;
-  margin-bottom: 12px;
+  font-size: $font-size-2xl;
+  line-height: 40px;
+  margin-bottom: 44px;
   @media screen and (max-width: $width-md) {
-    font-size: 22px;
-    line-height: 32px;
+    font-size: 28px;
+    line-height: 40px;
+    margin-bottom: 44px;
   }
 }
 
 .ds-h3,
 .ds-text-heading--xl {
-  margin-top: 31.5px;
-  margin-bottom: 10.5px;
+  font-size: $font-size-xl;
+  line-height: 32px;
+  margin-bottom: 32px;
   @media screen and (max-width: $width-md) {
-    font-size: 20px;
-    line-height: 25px;
+    font-size: $font-size-xl;
+    line-height: 32px;
+    margin-bottom: 32px;
   }
 }
 
 .ds-h4,
 .ds-text-heading--lg {
-  margin-top: 27px;
-  margin-bottom: 9px;
+  font-size: $font-size-lg;
+  line-height: 28px;
+  margin-bottom: 24px;
   @media screen and (max-width: $width-md) {
-    font-size: 18px;
-    line-height: 23px;
+    font-size: $font-size-lg;
+    line-height: 28px;
+    margin-bottom: 24px;
   }
 }
 
 .ds-h5,
 .ds-text-heading--md {
-  margin-top: 24px;
-  margin-bottom: 8px;
+  font-size: $font-size-md;
+  line-height: 24px;
+  margin-bottom: 16px;
   @media screen and (max-width: $width-md) {
-    font-size: 17px;
-    line-height: 21px;
+    font-size: $font-size-md;
+    line-height: 24px;
+    margin-bottom: 16px;
```

Comment:
I think you could use `$spacer-2` here from [the core layout variables](https://github.com/CMSgov/design-system/blob/development/packages/design-system/src/styles/settings/variables/_layout.scss#L15)